hazelcast-client/src/main/java/com/hazelcast/client/config/ClientReliableTopicConfig.java

```diff
@@ -69,9 +82,12 @@ public String getName() {
     /**
      * Sets the name or name pattern for this config. Must not be modified after this
      * instance is added to {@link ClientConfig}.
+     *
+     * @return the updated ClientReliableTopicConfig
      */
-    public void setName(String name) {
+    public ClientReliableTopicConfig setName(String name) {
```

Comment:
I was just writing a comment about this because I wasn't sure.

Reply:
I can get around this by using the method with nameSetter for all. 
```
 public static <T> T getConfig(ConfigPatternMatcher configPatternMatcher,
                                  Map<String, T> configs, String name,
                                  Class clazz, BiConsumer<T, String> nameSetter) 
```
Let me change it that way, so that we can discuss further if we want this prd or not. 

My main purpose is to make the behaviour of `client.getReliableTopic` aligned with all others. 
This prd is to standardize the way, so that we won't do the similar mistake again.